libs/dkg/src/dkg.cpp

```diff
@@ -198,6 +198,9 @@ void DistributedKeyGeneration::BroadcastQualCoefficients()
       static_cast<uint8_t>(State::WAITING_FOR_QUAL_SHARES), coefficients, "signature"}});
   complaints_answer_manager_.Clear();
   state_ = State::WAITING_FOR_QUAL_SHARES;
+  std::unique_lock<std::mutex> lock{mutex_};
```

Comment:
I would suggest to write it like this:

```
{
  unique_lock<> {mutex_}
  thing...
}
```

Since doing a manual unlock opens yourself to the chance of forgetting to unlock it at some point

Reply:
Thanks, I will make the change.